source/assets/js/classes/fef-responsive-helper.js

```diff
@@ -27,6 +27,15 @@ export class FefResponsiveHelper {
         return this.getBreakpoint() === 'desktop';
     }
 
+    /**
+     * Returns true if current breakpoint is 'desktop' or 'desktop-wide'
+     *
+     * @returns {boolean}
+     */
+    static isDesktopUp() {
+        return this.getBreakpoint() === 'desktop' || this.getBreakpoint() === 'desktop-wide';
```

Comment:
Cool, merci.

Noch etwas eleganter wäre vielleicht
`return this.isDesktop() || this.isDesktopWide();`

Bräuchte dann aber noch eine neue isDesktopWide()-Methode. Magst Du das noch machen? Sonst mach' ich das nachher in meinem Branch.

Reply:
Mach ich, aber die isDesktopWide haben wir schon 👍

Reply:
au, ja, hab' ich übersehen - um so besser 👍